lib/chef_apply/action/install_chef/minimum_chef_version.rb

```diff
@@ -17,12 +17,14 @@
 
 require "chef_apply/error"
 require "chef_apply/action/install_chef/minimum_chef_version"
+require_relative "../../dist"
 
 module ChefApply
   module Action
     class InstallChef < Base
       class MinimumChefVersion
 
+        D = ChefApply::Dist
```

Comment:
I feel this is a bit obscure but, I also don't like typing that much so, it is fine I guess. 😄

Reply:
Also, other parts of the codebase do it, so 👍

Reply:
all the `ChefApply::DIST::SOMEREALLYLONGSTRINGS` were making my eyes bleed o.O